language/libra-vm/src/libra_transaction_executor.rs

```diff
@@ -189,8 +189,8 @@ impl LibraVM {
                 .map_err(|e| e.into_vm_status())?;
             session
                 .execute_script(
-                    script.code().to_vec(),
-                    script.ty_args().to_vec(),
+                    script.code(),
```

Comment:
those changes are good. I think we had the script being deconstructed in the past - as I think it should - and the argument as `Vec<>` would be good then, but this is good no matter what, so I think we should steal and push it in master :)

Reply:
Cool!